src/process/ProcessExecution.js

```diff
@@ -102,7 +102,10 @@ export default function ProcessExecution(parentActivity, context) {
       }
     }
 
-    postponed.slice().forEach(({content}) => getActivityById(content.id).resume());
+    postponed.slice().forEach(({content}) => {
+      content = getActivityById(content.id);
```

Comment:
Try not to reassign the argument. It could be replaced with something like this:
```js
const activity = getActivityById(content.id);
if (activity) activity.resume();
```